src/iiiftilesource.js

```diff
@@ -374,17 +377,21 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
     /**
      * Determine whether arbitrary tile requests can be made against a service with the given profile
      * @function
-     * @param {object} profile - IIIF profile object
+     * @param {object} profile - IIIF profile array
```

Comment:
You should update `{object}` to `{array}`.

Reply:
right, I'll update the PR tomorrow